google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -126,6 +126,8 @@ appengine.flex.deployment.cost.warning.afterLink=for pricing information.
 appengine.flex.deployment.appyaml.edit.button.text=edit
 appengine.flex.deployment.docker.directory.label=Docker directory:
 appengine.flex.deployment.use.different.docker.directory=Use a different Docker directory
+appengine.flex.deployment.staged.artifact.name.label=Staged artifact name:
+appengine.flex.deployment.staged.artifact.name.tooltip=The filename to use when staging your artifact for deployment. Directories are allowed; all paths are relative to the temporary staging directory.
```

Comment:
> "Directories are allowed"? Can you clarify?

You can have something like "my/dir/with/my/artifact.war" if you want.

> Also, should probably have some wording to indicate that the Dockerfile artifact name should match up with this.

Yeah was trying to keep it short, but I'll add something in about that and reword it a bit.